bin/db_wrapper.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/python3
+
```

Comment:
I'd suggest this one to go under `binaryaudit`. This is not a runnable script but a module that would be then reused by other tools. The shebang will be then not needed and one could omit the `_abichecker_` part from the method names.

Thanks

Reply:
Fixed.